app/policies/application_policy.rb

```diff
@@ -1,53 +1,72 @@
 class ApplicationPolicy
+  include ApplicationScope
+
   attr_reader :user, :record
 
   def initialize(user, record)
     @user = user
     @record = record
   end
 
-  def index?
-    false
+  delegate :superadmin?, to: :user
+  delegate :department_manager?, to: :user
+  delegate :social_worker?, to: :user
+  delegate :volunteer?, to: :user
+
+  def superadmin_or_social_worker?
+    superadmin? || social_worker?
   end
 
   def show?
     scope.where(id: record.id).exists?
   end
 
-  def create?
-    false
+  def scope
+    Pundit.policy_scope!(user, record.class)
   end
 
-  def new?
-    create?
+  def user_is_in?
+    user.present?
   end
 
-  def update?
+  def deny_all?
     false
   end
 
-  def edit?
-    update?
+  def allow_all?
+    true
   end
```

Comment:
Maybe rename these methods `allow_all!` / `deny_all!` since they're not conditional?